src/component/front_end/ui/View.java

```diff
@@ -0,0 +1,20 @@
+package component.front_end.ui;
+
+
+abstract class View<T> implements ViewSpec {
+    protected final static String STRING_NEW_LINE = "\n";
```

Comment:
actually you can specify these behaviour in a separate method, and make all these constants `private`

```java
private static final String STRING_NEW_LINE = "\n";
private StringBuilder contentBuilder;
...

protected void addLine(String string) {
    contentBuilder.add(string);
    contentBuilder.add(STRING_NEW_LINE);
}
```

then after that, you can use this inside the child classes, without caring how the parent uses it:
```java
class ChildView extends View {
    public String getContent() {
         this.addLine("Hello World!");
    }
}
```